VAMobile/src/components/VAIcon/VAIcon.tsx

```diff
@@ -223,12 +225,32 @@ const VAIcon: FC<VAIconProps> = ({ name, width, height, fill, fill2, stroke, pre
     return <></>
   }
 
-  if (width && isFinite(width)) {
```

Comment:
While I like removing the if width/height checks in theory to reduce nesting, in practice I think it makes the code more  brittle since we're doing manipulation that only makes sense if they exist and effectively checking if they exist multiple times now with the scaled___ consts and then again with the final___ to actually override in iconProps. 

Coming back to this later seems more likely to cause a bug than it just being clear from the outset that the current structure does not make sense to scale or max scale unless width/height are explicitly set (since we're not reading the values from the SVG size).